include/emitc/emitc_mhlo.h

```diff
@@ -523,6 +523,25 @@ Dest slice(Src x, Tensor1D<int64_t, 2> start_indices,
   return z;
 }
 
+// Overload for 3d case
+template <typename Dest, typename Src, IsTensorOfDim<3, Src> = true>
+Dest slice(Src x, Tensor1D<int64_t, 3> start_indices,
+           Tensor1D<int64_t, 3> limit_indices, Tensor1D<int64_t, 3> strides) {
+  Dest z;
+
+  size_t index = 0;
+  for (int64_t i = start_indices[0]; i < limit_indices[0]; i += strides[0]) {
+    for (int64_t j = start_indices[1]; j < limit_indices[1]; j += strides[1]) {
+      for (int64_t k = start_indices[2]; k < limit_indices[2];
+           k += strides[2]) {
```

Comment:
Hum, I don't like the line break here, but our linter action does not complain...